### ISSUE TITLE
"Argument Error: unknown keyword" with yaml_safe_load with latest version of rubocop

### ISSUE THREAD

**Issue by jasnow** [rubocop-hq/rubocop#6502]
**TITLE: "Argument Error: unknown keyword" with yaml_safe_load with latest version of rubocop**

# Expected behavior

I expect that rubcop 0.60.0 "rake" run would be error free as it was for 0.59.2 gem.

## Actual behavior

Included the the "Steps to reproduce" section.

## Steps to reproduce the problem

 1. Versions
 * Ruby:    2.6.0-preview2
 * Rails:   5.2
 * Bundler: 1.17.1

 2. Similar to https://github.com/rubocop-hq/rubocop/issues/6412 but not using psych gem.

 3. This affects 11 repos of mine.

**STEPS TO REPRODUCE**
==================

 1. Clone https://github.com/jasnow/testing-app

 2. Run "rake" and see that it is green.

 3. Run "bundle update rubocop-rspec" to upgrade rubocop-rspec and robocop gems.
```
-    rubocop (0.59.2)
+    rubocop (0.60.0)
       jaro_winkler (~> 1.5.1)
       parallel (~> 1.10)
       parser (>= 2.5, != 2.5.1.1)
       powerpack (~> 0.1)
       rainbow (>= 2.2.2, < 4.0)
       ruby-progressbar (~> 1.7)
-      unicode-display_width (~> 1.0, >= 1.0.1)
-    rubocop-rspec (1.30.0)
-      rubocop (>= 0.58.0)
+      unicode-display_width (~> 1.4.0)
+    rubocop-rspec (1.30.1)
+      rubocop (>= 0.60.0)
```

 4. Run "rake" again and get this output:
```
rake aborted!
ArgumentError: unknown keywords: whitelist_classes, whitelist_symbols, aliases, filename
/Users/USER/.rvm/gems/ruby-2.6.0-preview2@ruby2.6-rails5.2/gems/rubocop-0.60.0/lib/rubocop/config_loader.rb:185:in `yaml_safe_load'
/Users/USER/.rvm/gems/ruby-2.6.0-preview2@ruby2.6-rails5.2/gems/rubocop-0.60.0/lib/rubocop/config_loader.rb:159:in `load_yaml_configuration'
/Users/USER/.rvm/gems/ruby-2.6.0-preview2@ruby2.6-rails5.2/gems/rubocop-rspec-1.30.1/lib/rubocop/rspec/inject.rb:8:in `defaults!'
/Users/USER/.rvm/gems/ruby-2.6.0-preview2@ruby2.6-rails5.2/gems/rubocop-rspec-1.30.1/lib/rubocop-rspec.rb:24:in `<top (required)>'
/Users/USER/.rvm/gems/ruby-2.6.0-preview2@ruby2.6-rails5.2/gems/bundler-1.17.1/lib/bundler/runtime.rb:81:in `require'
/Users/USER/.rvm/gems/ruby-2.6.0-preview2@ruby2.6-rails5.2/gems/bundler-1.17.1/lib/bundler/runtime.rb:81:in `block (2 levels) in require'
/Users/USER/.rvm/gems/ruby-2.6.0-preview2@ruby2.6-rails5.2/gems/bundler-1.17.1/lib/bundler/runtime.rb:76:in `each'
/Users/USER/.rvm/gems/ruby-2.6.0-preview2@ruby2.6-rails5.2/gems/bundler-1.17.1/lib/bundler/runtime.rb:76:in `block in require'
/Users/USER/.rvm/gems/ruby-2.6.0-preview2@ruby2.6-rails5.2/gems/bundler-1.17.1/lib/bundler/runtime.rb:65:in `each'
/Users/USER/.rvm/gems/ruby-2.6.0-preview2@ruby2.6-rails5.2/gems/bundler-1.17.1/lib/bundler/runtime.rb:65:in `require'
/Users/USER/.rvm/gems/ruby-2.6.0-preview2@ruby2.6-rails5.2/gems/bundler-1.17.1/lib/bundler.rb:114:in `require'
/Users/USER/Projects/testing-app/config/application.rb:15:in `<top (required)>'
/Users/USER/Projects/testing-app/Rakefile:4:in `<top (required)>'
/Users/USER/.rvm/gems/ruby-2.6.0-preview2@global/gems/rake-12.3.1/exe/rake:27:in `<top (required)>'
(See full trace by running task with --trace)
```
 5. Feel free to add a comment/question to this issue and I'll respond. Thanks.

**Comment by koic** [comment]
Thanks for the report. This issue is solved by #6396, #6413, and #6470.

> ArgumentError: unknown keywords: whitelist_classes, whitelist_symbols, aliases, filename

In ruby-head the keywords has been changed further, it is solved with #6470.

**Comment by jasnow** [comment]
@koic - When will this be on rubygems.org?

**Comment by jasnow** [comment]
> @koic - When will this be on rubygems.org?

Upgraded my gems and have verified the fixes. THANKS!!!

**Comment by joshuapinter** [comment]
> Now getting this for some of my repos: ArgumentError: unknown keywords: permitted_classes, permitted_symbols

This was fixed in version [0.61.0](https://github.com/rubocop-hq/rubocop/commit/4910866c4f0cd776564348bd13ebb7035e36904a).
